repository/s.json

```diff
@@ -2549,6 +2549,17 @@
 				}
 			]
 		},
+		{
+			"name": "SMPL Syntax",
+			"details": "https://github.com/ofekih/smpl-sublime",
+			"labels": ["smpl"],
```

Comment:
I would recommend adding the "language syntax" label here, and removing "Syntax" from the package name: by convention language support packages are simply named after the language because they usually do more than just the syntax like snippets and even the syntax itself unlocks various other features like highlighting but also autocomplete and goto definition.

Reply:
thanks for the feedback, done :)